lib/tasks/data_hygiene.rake

```diff
@@ -52,4 +52,36 @@ namespace :data_hygiene do
       speech.save(validate: false)
     end
   end
+
+  desc "Make attachments' command paper numbers valid"
```

Comment:
Minor: we should have something somewhere to remove (all) this when we're done, as it won't be used again.

Reply:
You mean just a note in the description? Like this?

```suggestion
  desc "Make attachments' command paper numbers valid (temporary. Will delete task after use)"
```

Reply:
Hmm...I was thinking more of a follow-up task, or a 'DO NOT MERGE' PR to remove it.

Reply:
Have now added this as a To-do item on the card.